L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -277,6 +277,14 @@ namespace trklet {
     void setStripLength_PS(double stripLength_PS) { stripLength_PS_ = stripLength_PS; }
     void setStripLength_2S(double stripLength_2S) { stripLength_2S_ = stripLength_2S; }
 
+    int numrinvbins() const { return numrinvbins_; }
+    void setNumrinvbins(int numrinvbins) { numrinvbins_ = numrinvbins; }
+
+    double overlapsize() const { return overlapsize_; }
+    void setOverlapsize(double overlapsize) { overlapsize_ = overlapsize; }
+
+    //Grabs the bin edges for the number of bins you need
```

Comment:
This is a vague comment. It should mention its for duplicate removal.

Reply:
Comment has been changed to say, `//Grabs the bin edges you need for duplicate removal bins`